python/paddle/fluid/layer_helper.py

```diff
@@ -33,13 +33,65 @@ class LayerHelper(object):
     def __init__(self, layer_type, **kwargs):
         self.kwargs = kwargs
         self.layer_type = layer_type
+        self._act = self.kwargs.get('act', None)
+        self._use_cudnn = self.kwargs.get('use_cudnn', None)
+        self._use_mkldnn = self.kwargs.get('use_mkldnn', None)
+        self._param_attr = ParamAttr._to_attr(
+            self.kwargs.get('param_attr', None))
+        self._bias_attr = ParamAttr._to_attr(self.kwargs.get('bias_attr', None))
+        self._param_name = None
+        self._input_param = []
         name = self.kwargs.get('name', None)
         # TODO(panyx0718, minqiyang): imperative mode
         # can not use both `layer_type` and `name`. Deprecate LayerHelper
         # and write a Helper for imperative mode.
         if name is None:
             self.kwargs['name'] = unique_name.generate(self.layer_type)
 
+    @property
+    def param_name(self):
+        return self._param_name
+
+    @param_name.setter
+    def param_name(self, pname):
+        self._param_name = pname
+
+    @property
+    def input_param(self):
+        return self._input_param
+
+    @input_param.setter
+    def input_param(self, input):
+        assert isinstance(input, [])
+        self._input_param = input
+
+    @property
+    def act(self):
+        return self._act
+
+    @act.setter
+    def act(self, activation):
+        assert (isinstance(activation, str) or (activation is None))
+        self._act = activation
+
+    @property
+    def use_cudnn(self):
```

Comment:
why add this?

Reply:
previous layer helper using `kwargs` to init all members, and now we need to init these members in __init__ to allow user change members after layer_helper object created